spec/unit/source/git_spec.rb

```diff
@@ -62,6 +62,38 @@
       expect(master_env.dirname).to eq 'master'
     end
   end
+
+  describe "generating environments without the ignore_branch_prefixes" do
```

Comment:
I think this test description is a little funky. When i first read it I presumes it would be a test without using the ignore_branch_prefixes setting. Maybe `describe "generate_environments respects ignore_branch_prefixes setting"`?

Reply:
Yeah, this has been through a few iterations to get here.

Reply:
Yeah this went through too many editing iterations.